plugins/openshift/README.md

```diff
@@ -0,0 +1,66 @@
+# servicemesh-plugin
+OpenShift Dynamic Plugin for ServiceMesh
+
+## Run Kiali and OpenShift Console in a local environment
+
+Install an OpenShift platform.
+
+This plugin is being developed using [Red Hat OpenShift Local](https://developers.redhat.com/products/openshift-local/overview) but any other plaform should work.
+
+Using CRC a local installation consists in:
+
+```
+crc stop
+crc cleanup
+crc start
+
+# Setup oc command if you don't have one
+eval $(crc oc-env)
+
+oc config use-context crc-admin
+
+# Install Istio on top of OpenShift
+$KIALI_HOME/hack/istio/install-istio-via-istioctl.sh
+
+# Install demos used for testing
+$KIALI_HOME/hack/istio/install-testing-demos.sh -c "oc"
+```
+
+In one terminal window, run Kiali in standalone and using the plugins flag:
+
+```
+cd $KIALI_HOME
+
+make build-plugin-openshift
```

Comment:
This fails for me:

```
$ make build-plugin-openshift 
yarn run v1.22.17
$ yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack
$ rm -rf dist
$ ts-node -O '{"module":"commonjs"}' node_modules/.bin/webpack
/bin/sh: ts-node: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [make/Makefile.build.mk:52: build-plugin-openshift] Error 127
```

Reply:
I thought maybe I need to do `yarn install` inside the `plugins/openshift` directory, and it appears, yes, that gets me this "ts_node" that is needed. But when I do that, git wants to commit changes that are made (should these go in .gitignore? Should I even do `yarn install` here?)
```
$ pwd
/home/jmazzite/source/kiali/kiali/plugins/openshift

$ git status
On branch lucasponce-plugins-structure
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	../../node_modules/
	../../yarn.lock

```